mlflow/shap.py

```diff
@@ -32,3 +41,124 @@ def _log_matplotlib_figure(fig, out_file, artifact_path=None):
     """
     with _log_artifact_contextmanager(out_file, artifact_path) as tmp_path:
         fig.savefig(tmp_path)
+
+
+@experimental
+def log_explanation(predict_function, features, artifact_path=None):
+    r"""
```

Comment:
Added `r` here to suppress the following sphinx error:

```
/home/circleci/project/mlflow/shap.py:docstring of mlflow.shap.log_explanation:82: WARNING: Explicit markup ends without a blank line; unexpected unindent.
```

https://app.circleci.com/pipelines/github/mlflow/mlflow/1652/workflows/c94ca5f6-9715-4840-9a32-46be18f5ba20/jobs/1654